contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpClientHandler.java

```diff
@@ -112,7 +113,7 @@ public HttpRequestContext handleStart(@Nullable Span parent, C carrier, Q reques
     }
     String spanName = getSpanName(request, extractor);
     SpanBuilder builder = tracer.spanBuilderWithExplicitParent(spanName, parent);
-    Span span = builder.startSpan();
+    Span span = builder.setSpanKind(Kind.CLIENT).startSpan();
```

Comment:
Do we also prefix the SpanName with "Recv." and "Sent." ?

Reply:
Asking because we should not do that.

Reply:
no we don't. I'll add that.

Reply:
Did you see my previous comment. We SHOULD NOT do that.

Reply:
for some reason it was showing only the first comment. Later I saw your comment and I edited my response. Anyway, I am not going to add the prefix.